Dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceDebugProcess.java

```diff
@@ -147,7 +150,6 @@ public void logError(final String message) {
           return;
         }
 
-        getSession().getConsoleView().print("Error: " + message + "\n", ConsoleViewContentType.ERROR_OUTPUT);
```

Comment:
Makes sense. I saw this frequently when working on this patch, but don't know if its something that users will encounter as much.

I can change the warn() to an error(), or add the print statement back in.

(re-tested: it looks like I still see it print out quite a bit: 

```
Error Response: {"code":-32602,"message":"Invalid params","data":{"request":{"method":"removeBreakpoint","params":{"isolateId":"isolates/47369335","breakpointId":"breakpoints/7"}},"details":"removeBreakpoint: invalid 'breakpointId' parameter: breakpoints/7"}}
```
)